go-controller/.golangci.yml

```diff
@@ -28,6 +29,23 @@ linters-settings:
     disable:
       - fieldalignment
       - shadow
+  
+  importas:
+    no-unaliased: true
+    alias:
+      # Kubernetes
+      - pkg: k8s.io/api/core/v1
+        alias: corev1
+      - pkg: k8s.io/apimachinery/pkg/apis/meta/v1
+        alias: metav1
+      - pkg: k8s.io/apimachinery/pkg/api/errors
+        alias: apierrors
+      - pkg: k8s.io/apimachinery/pkg/util/errors
+        alias: kerrors
```

Comment:
nit: kutilerrors

Reply:
agree that I care more about homogeneousness than what's its actually called. was just a nit